actionkit/api/orderrecurring.py

```diff
@@ -0,0 +1,195 @@
+from actionkit.api.base import ActionKitAPI
+
+from urllib.parse import urlencode
+
+class AKOrderRecurringAPI(ActionKitAPI):
+
+    def get_orderrecurring_detail(self, orderrecurring_id):
+        """
+            Get recurring donation info and billing info
+        """
+        if getattr(self.settings, 'AK_TEST', False):
+            return TEST_DATA.get('get_orderrecurring_detail')
+        result = self.client.get(
+            '%s/rest/v1/orderrecurring/%s' % (
+                self.base_url, orderrecurring_id))
+        rv = {'res': result}
+
+        if result.status_code == 200:
+            json = result.json()
+            rv.update(json)
+            order = json.get('order')
+            order_res = self.client.get('%s%s' % (self.base_url, order))
+            if order_res.status_code == 200:
+                order_json = order_res.json()
+                rv['order'] = order_json
+                user_detail = order_json.get('user_detail')
+                user_detail_res = self.client.get('%s%s' % (self.base_url, user_detail))
+                if user_detail_res.status_code == 200:
+                    rv['user_detail'] = user_detail_res.json()
+        return rv
+
+
+    def list_orderrecurring(self, user_id=False, query_params={}):
+        if getattr(self.settings, 'AK_TEST', False):
+            return TEST_DATA.get('list_orderrecurring')
+        if user_id:
+            result = self.client.get(
+                '%s/rest/v1/orderrecurring/?user=%s&%s' % (
+                    self.base_url, user_id, urlencode(query_params)))
+        else:
+            result = self.client.get(
+                '%s/rest/v1/orderrecurring/?%s' % (
+                    self.base_url, urlencode(query_params)))
+        rv = {'res': result, 'objects': []}
+        paginate = True
```

Comment:
(Optional) This is more a code-style thing, and your code below looks good.  I think the style can be tightened a bit by leveraging the default value in dict.gets, e.g.
```
while result.status_code == 200:
  json = result.json()
  rv['objects'].extend(json.get('objects', []))
  next_page = json.get('meta', {}).get('next', None)
  if not next_page:
       break
  result = self.client.get('%s%s' % (self.base_url, next_page))
```